torch/csrc/utils/pybind.cpp

```diff
@@ -7,8 +7,14 @@ namespace detail {
 
 bool type_caster<c10::SymInt>::load(py::handle src, bool) {
   if (torch::is_symint(src)) {
+    auto node = src.attr("node");
+    if (py::isinstance<c10::SymNodeImpl>(node)) {
+      value = c10::SymInt(py::cast<c10::SymNode>(node));
+      return true;
+    }
+
```

Comment:
Outside the if statement, what would be the type of `node` if `py::isinstance<c10::SymNodeImpl>(node)` evaluate to false?